FMUv1/README.md

```diff
@@ -0,0 +1,33 @@
+# FMUv1 Pixhawk Open Hardware Design
```

Comment:
FYI, minimal entry point, mostly to show:
- This design end of life, don't use it
- Explain what the files are and how you open them
- Feature overview is here because there is no other info remaining about this board. For other boards I will link to that info in PX4 user guide. 
- Link to licencing info.